airflow/providers/docker/operators/docker.py

```diff
@@ -318,18 +321,20 @@ def _get_cli(self) -> APIClient:
             tls_config = self.__get_tls_config()
             return APIClient(base_url=self.docker_url, version=self.api_version, tls=tls_config)
 
-    def get_command(self) -> Union[List[str], str]:
+    @staticmethod
+    def get_command(command: Union[str, List[str]]) -> Union[List[str], str]:
```

Comment:
It feels quite weird for me that `self.get_command()` is not internally dependent on `self.command`. Maybe it’d be a good idea to rename this to e.g. `format_command()` or `as_commands()`.

As an alternative, maybe it’d be reasonable to have *two* functions, one do the formatting (replacing the new `get_command()` function in this PR), and another that calls the formatting function to format `self.command` (like the previous `get_command()` implementation)?

Reply:
agree, `format_command` sounds better